### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Veaxx**
# Description

Please include a summary of the change, which issue is fixed and the context if the new feature. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant and add new options if necessary. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by BjornVanBruchem — CHANGES_REQUESTED** [review]
Yo mama

**Review by GXZheng312 — APPROVED** [review]

**Review by BjornVanBruchem — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Project Moesland
```
Comment (xandermonk) [inline README.md:2]: delete unnecessary whitespace!

**Review by xandermonk — CHANGES_REQUESTED** [review]

**Review by KevinVanH — APPROVED** [review]

**Review by xandermonk — APPROVED** [review]
